knowledge-base/modify-html5-viewer-area-width-through-kendo-splitter.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Modify default Document Map and Parameter areas width in Html5 Report Viewer
+description: Resize the default areas sizes through the splitter
+type: how-to
+page_title: Change Side areas sizes in Html5 Viewer with the Kendo splitter
+slug: modify-html5-viewer-area-width-through-kendo-splitter
+position: 
+tags: 
+ticketid: 1398733
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>13.0.19.116 and above</td>
+	</tr>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+	<tr>
+		<td>Report Viewer</td>
+		<td>all Html5 Report Viewers</td>
+	</tr>
+</table>
+
+
+## Description
+Sometimes it may be necessary to increase/decrease the width of the Document Map and Parameters Area inside report viewer.
```

Comment:
I think it would be better to specify here that this is an article which elaborates how to set the default width of those areas on initial load. Their width could be changed by splitter release handlers.